docs/manual/java/gettingstarted/understand-hello.md

```diff
@@ -0,0 +1,28 @@
+# Understanding Hello World
+
+After creating and running Hello World from the command line, you no doubt appreciate what Lagom framework did for you. There was no need to determine what infrastructure you might need and then install and configure it. The template removed the necessity to set up a project or build structure. And, as you create services of your own, Lagom detects changes and performs a hot reload! Lagom allows you to concentrate on satisfying your business needs.
+
+The separation of concerns illustrated in Hello World and an introduction to service descriptors and the registry will help you as you start developing your own microservices:
+
+* [Service interface](#service-interface)
+* [Service implementation](#service-implementation)
+
+## Service interface
+The source file defining a service interface belongs in the service's api project.  For example, in Hello World, `HelloService.java`, the source file for the `hello` service interface resides in the `hello-api` directory of the Maven project or sbt build.
+
+@[helloservice-interface](code/docs/javadsl/gettingstarted/helloservice/HelloService.java)
+
+Note that:
+ 
+* The service interface inherits from [`Service`](api/index.html?com/lightbend/lagom/javadsl/api/Service.html) and provides an implementation of [`Service.descriptor`](api/index.html?com/lightbend/lagom/javadsl/api/Service.html#descriptor--) method.
+
+* The implementation of `Service.descriptor` returns a [`Descriptor`](api/index.html?com/lightbend/lagom/javadsl/api/Descriptor.html). A `Descriptor` defines the service name and the REST endpoints offered by a service. For each declared endpoint, declare an abstract method in the service interface, as illustrated in the `HelloService.hello` method. For more information, see [[Service Descriptors|ServiceDescriptors]].
+
```

Comment:
This description of a `Decriptor` only covers a subset of what a Descriptor is. I would reword to "The `HelloService` descriptor defines the service name and a rest endpoint. [...] For more information and other features in the Descriptor API see `Service Descriptors`"